etl/models/data_files_config.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+import json
+from dataclasses import dataclass
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from dagster import ConfigurableResource, EnvVar
+
+if TYPE_CHECKING:
+    from etl.models.types import DataFileName
+
+
+class DataFilesConfig(ConfigurableResource):  # type: ignore[misc]
+    """
+    A ConfigurableResource that holds the Paths of data files.
```

Comment:
The naming here is ambiguous. These are input data files? In contrast to the output configuration?

Reply:
Yes, this class is for input data files only, so I renamed it to `InputDataFilesConfig`.